src/libraries/System.Drawing.Common/src/System/Drawing/Gdiplus.cs

```diff
@@ -367,7 +367,7 @@ public const int
         public const int ERROR_CANCELLED = 1223;
 
         [StructLayout(LayoutKind.Sequential)]
-        public class ENHMETAHEADER
+        public struct ENHMETAHEADER
```

Comment:
I think you can also change `public int nSize = 40; // ndirect.DllLib.sizeOf( this )` to just `public int nSize;`. The structure is not initialized anywhere so it is dead code; and the size of this structure is not 40.